files/zh-cn/web/api/webglrenderingcontext/readpixels/index.md

```diff
@@ -0,0 +1,123 @@
+---
+title: WebGLRenderingContext.readPixels()
+slug: Web/API/WebGLRenderingContext/readPixels
+---
+
+{{APIRef("WebGL")}}
+
+[WebGL API](/zh-CN/docs/Web/API/WebGL_API)的 **`WebGLRenderingContext.readPixels()`** 方法从当前color framebuffer中读取指定矩形的像素矩阵并转换为 {{jsxref("TypedArray")}} 或 {{jsxref("DataView")}} 对象。
```

Comment:
space between Chinese and English: <https://github.com/mdn/translated-content/blob/main/docs/zh-cn/translation-guide.md#%E4%B8%AD%E6%96%87%E5%92%8C%E6%8B%89%E4%B8%81%E8%AF%AD%E7%B3%BB%E6%96%87%E5%AD%97%E9%97%B4%E5%8A%A0%E7%A9%BA%E6%A0%BC>